README.md

```diff
@@ -451,40 +444,24 @@ See `docs/CHANGELOG.md`.
 
 ## Background
 
+### What's the plan and why?
+
+**Tuesday, September 19, 2017:** This was forked from [Bats][bats-orig] at
+commit [0360811][].  It was created via `git clone --bare` and `git push
+--mirror`. See the [Background](#background) section above for more information.
+
+[bats-orig]: https://github.com/sstephenson/bats
+[0360811]: https://github.com/sstephenson/bats/commit/03608115df2071fff4eaaff1605768c275e5f81f
+
+This [bats-core repo](https://github.com/bats-core/bats-core) is now the official Bats project.
+
 ### Why was this fork created?
 
-The original Bats repository needed new maintainers, and has not been actively
+The original Bats repository needed new maintainers, and had not been actively
```

Comment:
I would recommend dropping this particular change.  This adjustment to wording makes it sound like the original bats project is being actively maintained by the original maintainer again, when that is not the case.

Reply:
Although it's written in the past tense and sounds clear enough IMHO, perhaps "is no longer maintained" would dispel any ambiguity.